### DISCUSSION TITLE
SMTP Problems

### DISCUSSION THREAD

**Discussion by neils1108** [mastodon/mastodon#31664] [Q&A]
I tried doing a mastadon instance but it won't send confirmation mails. I will send you the .env.production contents so maybe someone can help me fix this issue. I use brevo for smtp.
![image](https://github.com/user-attachments/assets/dafdc5e7-daf6-4644-be09-1bf631197f54)

**Comment by aksthelion** [comment]
Does the smtp itself work? Have you tested it separately?
